dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNVoltageLevelInterfaceIIDM.cpp

```diff
@@ -65,6 +65,15 @@ voltageLevelIIDM_(voltageLevel) {
       graph_.addEdge(node1, node2, itSwitch.getId());
       weights1_[itSwitch.getId()] = 1;
     }
+    // Additional edges for internal connections
+    for (const powsybl::iidm::node_breaker_view::InternalConnection& itInternalConnection : voltageLevelIIDM_.getNodeBreakerView().getInternalConnections()) {
```

Comment:
Could we add a unit test for this part?

Reply:
Yes, we have validated the changes using a test case derived from IEEE 118 and published by ENTSO-E. It has been obtained using PowSyBl to import the original CIM/CGMES files provided by ENTSO-E in its CGMES [Conformity Assessment Scheme](https://www.entsoe.eu/digital/cim/cim-conformity-and-interoperability/). The IIDM defines the Network at node/breaker level and contains internal connections (not only switches). Furthermore, a DynaFlow run on it makes an overloaded line trip, allowing to test the update of the network topology after the time domain simulation. Do you think we could use this as the test case to check the node/breaker support?

Reply:
@marcochiaramello I'll let you comment on if this test can be used.

A test would be very good. Initially I was thinking about unit tests similar to what can be found in dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/* in which we build in c++ a specific small network to test that the internals connections are properly taken into account.

Reply:
about the IEEE118 test case, it can be an interesting one, if think we can add it!

Reply:
I have expanded the node/breaker network created in the unit tests of the powsybl IIDM data interface to add internal connections. Now the unit tests are checking the fixes made in `connectNode`, `disconnectNode` and `isNodeConnected` of the `VoltageLevelInterfaceIIDM` to support node/breaker topologies that include internal connections.